exercises/etl/canonical-data.json

```diff
@@ -39,8 +39,8 @@
         },
         "expected": {
           "a" : 1,
-          "e" : 1,
           "d" : 2,
+          "e" : 1,
           "g" : 2
         }
```

Comment:
Property's ordering has no meaning in JSON, but as this was the only object not *key-sorted*, it makes sense to sort it to keep consistency.